Adds readme files for every mv3 sample (#973)
GoogleChrome/chrome-extensions-samples: api-samples/override/blank_ntp/README.md

# "chrome_url_overrides"

> This sample demonstrates using the ` "chrome_url_overrides"` manifest key to replace the user's default New Tab page with a new html file.

Reviewer: ```suggestion
This sample demonstrates the ` "chrome_url_overrides"` manifest key by replacing the user's default New Tab page with a new html file.
```
